src/analysis/normed/group/pointwise.lean

```diff
@@ -3,16 +3,117 @@ Copyright (c) 2021 Sébastien Gouëzel. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Sébastien Gouëzel
 -/
-import analysis.normed.group.basic
+import analysis.convex.basic
+import analysis.normed.group.add_torsor
 import topology.metric_space.hausdorff_distance
 
 /-!
-# Properties of pointwise addition of sets in normed groups.
+# Properties of pointwise addition of sets in normed groups
 
 We explore the relationships between pointwise addition of sets in normed groups, and the norm.
 Notably, we show that the sum of bounded sets remain bounded.
 -/
 
+section
+variables {α : Type*} [ordered_semiring α] {a b : α}
+
+lemma mul_lt_of_lt_one_left (hb : 0 < b) (ha : a < 1) : a * b < b :=
+(mul_lt_mul_of_pos_right ha hb).trans_le (one_mul _).le
+
+lemma mul_lt_of_lt_one_right (ha : 0 < a) (hb : b < 1) : a * b < a :=
+(mul_lt_mul_of_pos_left hb ha).trans_le (mul_one _).le
+
+end
+
+namespace emetric
+variables {α : Type*} [pseudo_emetric_space α] {δ ε : ℝ} {s : set α} {x y : α}
+
+lemma inf_edist_le_edist_add_inf_edist : inf_edist x s ≤ edist x y + inf_edist y s :=
+by { rw add_comm, exact inf_edist_le_inf_edist_add_edist }
```

Comment:
could you move this lemma to a better place?

Reply:
Already in mathlib :)